src/js/components/RadioButton/RadioButton.js

```diff
@@ -96,10 +101,10 @@ const RadioButton = forwardRef(
             })}
           />
           {children ? (
-            children({ checked, focus: focus && focusIndicator, hover })
+            children({ checked, focus: isFocused && focusIndicator, hover })
           ) : (
             <StyledRadioButtonBox
-              focus={focus && focusIndicator}
+              focus={isFocused}
```

Comment:
In RadioButtonGroup we pass down a `focus` prop to RadioButton, it seems like that props is no longer being used and so what gets passed from RadioButtonGroup is no longer bbeing taken into account for the focus